sdks/python/apache_beam/io/gcp/bigquery_tools.py

```diff
@@ -925,6 +925,12 @@ def __iter__(self):
       if self.schema is None:
         self.schema = schema
       for row in rows:
+        # return base64 encoded bytes as byte type on python 3
+        # to match behavior DataflowRunner
```

Comment:
Let's say "which matches the behavior of Beam Java SDK". Note that this does not depend on Py3.